docs/documentation/openapi.md

```diff
@@ -171,22 +196,3 @@ correct.
   Markdown format options like inline code \` (backtick) and `#`
   headings.
 
-* A single `|` (pipe) character begins a multi-line description on the
-  next line.  Single spaced lines (one newline at the end of each) are
-  joined. Use an extra blank line for a paragraph break.
-
-### Examples:
-
-```
-Description: This is a single line description.
-```
-
-```
-Description: |
-             This description has multiple lines.
-             Sometimes descriptions can go on for
-             several sentences.
-
-             A description might have multiple paragraphs
-             as well.
-```
```

Comment:
Why did you remove this?

Reply:
@timabbott since we have not used pipe character for multiline description in zulip.yaml, this syntax was only used in deprecated zulip-2.0.yaml, so to prevent confusion i removed this part.

Reply:
@timabbott should i add the part that you have mentioned?